man/set2key.Rd

```diff
@@ -4,6 +4,7 @@
 \alias{key2}
 \alias{key<-}
 \title{ Deprecated. }
+\keyword{internal}
```

Comment:
this is for `pkgdown` to skip this in the Reference index, right?

Reply:
Yeah, plus it excludes it from the regular package index (which I think is good practice).